mw-webapp/src/component/image/Image.tsx

```diff
@@ -1,11 +1,17 @@
 import clsx from "clsx";
+import {Cy, Modal} from "src/component/modal/Modal";
 import styles from "src/component/image/Image.module.scss";
 
 /**
  * Props for the Image component
  */
 interface ImageProps {
 
+  /**
+   * Data attributes for cypress testing
+   */
+  cy?: Cy;
```

Comment:
too much data attribute on the top level. Check component confirm.tsx for example. Keep the only ine property for testing purposes.

Reply:
I left only cy at the top level, creating a CyDataImage like Confirm.tsx as an example, and also added MODAL_CY to the IMAGE_CY variable in Image.cy.tsx to use only one testing property.